2022/CVE-2022-20699.json

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
Expected location (2022/20xxx/CVE-2022-20699.json) of file does not match actual location (2022/CVE-2022-20699.json)